modules/capi-troubleshooting.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * machine_management/capi-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="capi-troubleshooting_{context}"]
+= Troubleshooting clusters that use the Cluster API
+
+Use the information in this section to understand and recover from issues you might encounter. Generally, troubleshooting steps for problems with the Cluster API are similar to those steps for problems with the Machine API. 
+
+The Cluster CAPI Operator and its operands are provisioned in the `openshift-cluster-api` namespace, whereas the Machine API uses the `openshift-machine-api` namespace. When using `oc` commands that reference a namespace, be sure to reference the correct one.
+
+[id="ts-capi-cli_{context}"]
+== CLI commands return Cluster API machines
+
+For clusters that use the Cluster API, `oc` commands such as `oc get machine` return results for Cluster API machines. Because the letter `c` precedes the letter `m` alphabetically, Cluster API machines appear in the return before Machine API machines do. 
+
+* To list only Machine API machines, use the fully qualified name `machines.machine.openshift.io` when running the `oc get machine` command:
```

Comment:
Do you think it worths mentioning that the fully qualified name for CAPI is `machines.cluster.x-k8s.io`?

Reply:
yep, very good idea :+1: